tpot/operator_utils.py

```diff
@@ -243,7 +243,7 @@ def export(cls, *args):
                 # To make sure the inital operators is the first parameter just
                 # for better persentation
                 for dep_op_pname, dep_op_str in dep_op_list.items():
-                    if dep_op_str == 'f_classif':
+                    if dep_op_str in ['f_classif', 'f_regression']:
```

Comment:
Because `SelecPercentile` do not take `f_classif()` or `f_regression()`
```
In [3]: selector = SelectPercentile(f_classif, percentile=10)
   ...: selector.fit(X, y)
   ...:
Out[3]:
SelectPercentile(percentile=10,
         score_func=<function f_classif at 0x000002A3C66CF6A8>)

In [4]: selector = SelectPercentile(f_classif(), percentile=10)
   ...: selector.fit(X, y)
   ...:
   ...:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-bba9fbc03e91> in <module>()
----> 1 selector = SelectPercentile(f_classif(), percentile=10)
      2 selector.fit(X, y)
      3

TypeError: f_classif() missing 2 required positional arguments: 'X' and 'y'
```